tests/e2e-tests/config/default.json

```diff
@@ -1,3 +1,13 @@
 {
-  "url": "https://example.com/"
+  "url": "http://example.com",
```

Comment:
Perhaps we can change this file to have the correct defaults already in place? Eliminates a step to copy and change a file, I like the idea of this file being overridable in case you have a different setup and want to change things a bit though so lets keep that part, just populate this file with the correct default values as defined in the docker files.

Reply:
I thought of doing the same and first, have tried it without local `test:e2e.json` file. But when running the tests, I am getting the following warnings in the terminal:

```
console.error node_modules/config/lib/config.js:1425
    WARNING: NODE_ENV value of 'test:e2e' did not match any deployment config file names.

  console.error node_modules/config/lib/config.js:1426
    WARNING: See https://github.com/lorenwest/node-config/wiki/Strict-Mode
```

If I open the [documentation link provided in the warning](https://github.com/lorenwest/node-config/wiki/Strict-Mode), I am getting the following explanation and instructions on how to fix the warning:

> If NODE_ENV is set in the environment, we check if there is any file matching that name in the config/ directory containing valid syntax. You can create a file with the name matching the desired NODE_ENV value: {NODE_ENV}.json and include the contents '{}', which is an empty valid JSON object. If you are using YAML based config files, the file {NODE_ENV}.yml must not be empty, place a comment or any valid YAML syntax in the file.

And so I created `test:e2e.json`. We can technically leave it empty with the contents '{}' - which I tested and it works (given the fact that `default.json` has all the details we need). 

Maybe there is a better way to address the warning and I am missing something simple?

In any case, I also like the idea of keeping `default.json` with the details of the site and user built in Docker. I updated the `default.json` to reflect that.

Reply:
I also committed empty `test:e2e.json` so that it could be used in case the default values need to be overridden.